packages/core/src/components/menu/menuItem.tsx

```diff
@@ -263,6 +263,11 @@ export const MenuItem: React.FC<MenuItemProps> = React.forwardRef<HTMLLIElement,
             ...(disabled ? DISABLED_PROPS : {}),
             className: anchorClasses,
         },
+        isSelected ? (
+            <span className={Classes.MENU_ITEM_SELECTED_ICON}>
```

Comment:
can this class be applied directly to `<SmallTick />`? we might be able to collapse this and remove the extra container element.